pkg/controllers/management/catalog/controller.go

```diff
@@ -16,7 +15,7 @@ import (
 
 func Register(ctx context.Context, management *config.ManagementContext) {
```

Comment:
Looks like this is still running in master controller, which is only running in a single leader(so cache will only exist inside leader instance) @erikwilson have you tested git catalog on a HA setup?

Reply:
wait... I get it now.